tools/gn

```diff
@@ -134,7 +136,15 @@ def to_gn_args(args):
       gn_args['use_goma'] = False
       gn_args['goma_dir'] = None
 
-    gn_args['use_glfw'] = args.use_glfw
+    # Only attempt to use ccache if goma is disabled or unavailable.
+    if gn_args['use_goma']:
+      gn_args['use_ccache'] = False
+    else:
+      ccache_binary = spawn.find_executable('ccache')
```

Comment:
I'd make the call to `spawn.find_executable` only if `args.ccache`:

`if args.ccache and spawn.find_executable('ccache'):`